get_lims_info.py

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env python
-
-from genologics import lims
-from config import URI, USER, PASSWORD
```

Comment:
Maybe you want to read this from a config file that can be places anywhere and distributed separately?